homeassistant/components/sensor/const.py

```diff
@@ -487,7 +487,8 @@ class SensorStateClass(StrEnum):
         UnitOfTemperature.FAHRENHEIT,
     },
     SensorDeviceClass.VOLATILE_ORGANIC_COMPOUNDS: {
-        CONCENTRATION_MICROGRAMS_PER_CUBIC_METER
+        CONCENTRATION_MICROGRAMS_PER_CUBIC_METER,
+        CONCENTRATION_PARTS_PER_MILLION,
```

Comment:
ppb is parts per **billion**, not _million_. Is this intentional?

Reply:
Not really. But this pull is going no where. A new device class needs to be added.